python/ray/air/tests/test_tracebacks.py

```diff
@@ -95,7 +95,7 @@ def failing(config):
 
     tuner = Tuner(failing)
     results = tuner.fit()
-    assert len(str(results[0].error).split("\n")) <= 12
+    assert len(str(results[0].error).split("\n")) <= 20
```

Comment:
what happened here?

Reply:
py312 adds a bunch of new lines + formatting to stack traces 😆